azure_sdk/lib/v2019_03_01/modules/authorization_profile_module.rb

```diff
@@ -0,0 +1,133 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require 'azure_mgmt_authorization'
+
+module Azure::Profiles::V2019_03_01
```

Comment:
Here Azure::Profiles::V2019_03_01 should be Azure::Profiles::V2019_03_01_hybrid ? Checkign with Vishnu on the usage of other SDKs

Reply:
We have the _hybrid for Java/C#/Python SDKs, but not for Go and Ruby. @sarangan12 going forward can we append with _hybrid

Reply:
Done